subsys/usb/class/cdc_acm.c

```diff
@@ -67,23 +67,16 @@ LOG_MODULE_REGISTER(usb_cdc_acm);
 /* Size of the internal buffer used for storing received data */
 #define CDC_ACM_BUFFER_SIZE (2 * CONFIG_CDC_ACM_BULK_EP_MPS)
 
-
 /* Max CDC ACM class request max data size */
 #define CDC_CLASS_REQ_MAX_DATA_SIZE	8
 
 /* Serial state notification timeout */
 #define CDC_CONTROL_SERIAL_STATE_TIMEOUT_US 100000
 
-#define CDC_ACM_INT_EP_ADDR		0x85
```

Comment:
Can you please merge it to https://github.com/zephyrproject-rtos/zephyr/pull/13310/commits/3bf084095de588c6caf2915d23eb58553dc9d55d and use the same schema for AUTO_EP_IN, AUTO_EP_INT, AUTO_EP_OUT as in  `subsys/usb/class/hid/core.c` below.